packages/slice-machine/tests/server/services/getPrismicData.test.ts

```diff
@@ -23,42 +22,21 @@ describe("getPrismicData", () => {
       os.homedir()
     );
 
-    const result = await getPrismicData(base);
+    const result = getPrismicData();
     expect(result.isOk()).toBeTruthy();
-    expect(result.isOk() && result.value.base).toEqual(base);
```

Comment:
what's base going to be now?

Reply:
getPrismicData doesn't return a base anymore.

Reply:
So the base has to be retrieved from the SM.json like you did